cmd/codecrafters/main.go

```diff
@@ -26,11 +26,13 @@ EXAMPLES
   $ codecrafters test              # Run tests without committing changes
   $ codecrafters test --previous   # Run tests for the current stage and all previous stages without committing changes
   $ codecrafters submit            # Commit changes & submit to move to next step
+  $ codecrafters update-buildpack  # Update buildpack configuration 
```

Comment:
Let's remove this from examples - not a commonly used command and doesn't need to be highlighted